apps/shipments/views.py

```diff
@@ -48,6 +48,8 @@ def create(self, request, *args, **kwargs):
         # Create Shipment
         serializer = ShipmentCreateSerializer(data=request.data, context={'auth': request.auth})
         serializer.is_valid(raise_exception=True)
+        print('Request.ath: ', request.auth)
+        serializer.validate_wallet_ownership(request.data['shipper_wallet_id'])
```

Comment:
I don't think this check belongs as a separate call on the ShipmentCreateSerializer. Check out the docs on field level validation in serializers: https://www.django-rest-framework.org/api-guide/serializers/#field-level-validation